lib/inc/internal/simple_config_origin.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <hocon/config_origin.hpp>
+#include <hocon/config_value.hpp>
```

Comment:
Ah, looks like I added it because of one of the implementations of `simple_config_origin::merge_origins`, with:

```
    shared_origin simple_config_origin::merge_origins(std::vector<shared_value> const& stack) {
        vector<shared_origin> origins;
        for (auto& v : stack) {
            origins.push_back(v->origin());
        }
```
If I didn't include it, I'd get 'error: member access into incomplete type 'element_type' (aka 'const hocon::config_value') origins.push_back(v->origin());`